catalog/dags/common/sql.py

```diff
@@ -172,3 +182,25 @@ def get_db_hook(self):
             postgres_conn_id=self.postgres_conn_id,
             **self.hook_params,
         )
+
+
+@task
+def run_sql(
+    postgres_conn_id: str,
+    sql_template: str,
+    task: AbstractOperator = None,
+    timeout: float = None,
+    handler: callable = RETURN_ROW_COUNT,
+    **kwargs,
+):
+    # TODO this can potentially be used in other DAGs (batched_update, delete_records, add_license_url)
```

Comment:
I have a note in my TODOs to make an issue for this 😄 I'll take care of that soon! haha